examples_RPi/RF24Mesh_Example_Master.py

```diff
@@ -20,5 +22,8 @@
     mesh.DHCP()
 
     while network.available():
-        print("Received message")
         header, payload = network.read(10)
+        if chr(header.type) == 'M':
+            print("Rcv {} from 0{:o}".format(unpack("L",payload)[0], header.from_node))
```

Comment:
At my environment this doesn't work cause the unpack methods requires a string of 4 letters. So to fix that you just need:
`print("Rcv {} from 0{:o}".format(unpack(('L'),payload[:calcsize("L")])[0], header.from_node))`
For this you also need to include this method from the struct-package -> 
`from struct import unpack,calcsize`

Reply:
How big is your payload? Slave example sends uint32_t, which is 4bytes long?

Which version of python you are using? I'm using only Python 3. Python 2 is bit mysterious for me :-D

Thanks for revision, I'm ready to push a correction

Reply:
Well I think the problem could be the python 3 / 2 ... But when I start this script with python3 it doesn't even start (It has problem with finding the imports and with running the modules ...) .  So when I run in python2 this is the only mistake.

Reply:
Ok, I will try your suggested correction in my instalation as soon as possible, and then push it here.

Thank you